sdk/ml/azure-ai-ml/tests/pipeline_job/unittests/test_pipeline_job_entity.py

```diff
@@ -206,16 +205,17 @@ def test_command_job_with_invalid_mode_type_in_pipeline_deserialize(self):
             "hello_python_world_job": {
                 "inputs": {
                     "sample_input_data": {
+                        'mode': 'ro_mount',
```

Comment:
Nit: is it possible to use unified "" here?

Reply:
unified